images/docker-compose-opcua.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  ioc:
+    image: ghcr.io/cnpem/opcua-epics-ioc
+    build:
+      context: ./
+      dockerfile: ../Dockerfile
+      target: no-build
+      labels:
+        org.opencontainers.image.source: https://github.com/cnpem/epics-in-docker
+      args:
+        REPONAME: opcua
+        RUNDIR: /opt/epics/modules/opcua
```

Comment:
If we are going to use #59, we should be careful when defining these variables. By implementing the suggestion of dropping the `RUNDIR` as input to keep binaries, `REPONAME` should point to `epics/modules/opcua` instead. There is no such `/opt/opcua` directory in `no-build` targets as of now.

An alternative approach could be to symbolic link or hard-copy during the pruning stage of `no-build`.